ComposureWatch/src/Components/SVG/CheckMarkSvg.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+const CheckMarkSvg = () => {
```

Comment:
Does it being named SVG matter? What if I decided to change it from a svg to a png because lets say png becomes the new thing, would that make the naming kinda weird?

What if you named it something like `CheckmarkIcon`? That way you are telling it what you want, not necessarily how you want it.